frontend/src/app/coworking/room-reservation/reservation-table.service.ts

```diff
@@ -85,19 +96,47 @@ export class ReservationTableService {
     );
   }
 
+  getRoomInformation(): Observable<RoomDetails[]> {
+    return this.http.get<RoomDetails[]>(`/api/coworking/rooms/`);
+  }
```

Comment:
Instead of adding it to this service, it would be better to inject the academics service, which has a call for using rooms. Better yet, since rooms is being used in many places, it would probably be more ideal for us to create a completely separate `RoomService` that is injected into all these places - perhaps we can make a ticket for this.

Reply:
Looping in @aarjavjain2002